The imageioapphelpers library is static and so we do not need to distribute it.

--- OpenColorIO-2.4.0/src/libutils/imageioapphelpers/CMakeLists.txt.orig
+++ OpenColorIO-2.4.0/src/libutils/imageioapphelpers/CMakeLists.txt
@@ -39,12 +39,3 @@
         OpenColorIO
         utils::strings
 )
-
-install(TARGETS imageioapphelpers
-    LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
-    ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
-)
-
-install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/imageio.h
-    DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/OpenColorIO/imageioapphelpers
-)
\ No newline at end of file
